### ISSUE TITLE
Scroll to view if not present Appium

### ISSUE THREAD

**Issue by nathanchen13** [TranslationWMcs435/TranslationWMcs435#78]
Add this so that if a button or other view is not present, it scrolls to it.

**Comment by King-AK** [comment]
Im not sure if this is going to be possible since Appium is blackbox testing and can only reference whats present on the screen as far as I know. I will look into it though.

**Comment by MarkHutchens** [comment]
Does Appium throw any sort of error if it can't click on something? UIAutomator does, and I'm bracing myself for surrounding every click with a try/catch block. (A pretty terrible solution, I admit. Probably won't do it this sprint.)

**Comment by King-AK** [comment]
Yeah when appium cannot click/perform the desired action on something it
throws this error:
"An element could not be located on the page using the given search
parameters..."

**Comment by King-AK** [comment]
Appium is scrolling like a boss now

**Comment by King-AK** [comment]
Well, actually its scrolling on the test file.

**Comment by MarkHutchens** [comment]
What, not on anything else?
